src/index.js

```diff
@@ -0,0 +1,16 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import App from './App';
+import store from './redux/configureStore';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <App />
+    </Provider>
+  </React.StrictMode>,
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=NOTE%3A%20Make%20sure%20you%20only%20dispatch%20those%20actions%20once%20and%20do%20not%20add%20data%20to%20store%20on%20every%20re%2Drender%20(i.e.%20when%20changing%20views%20/%20using%20navigation).), Please make sure that every action dispatched once because now when I navigate rockets or missions pages all actions are dispatched twice and that's happening because react strict mode is enabled.

Reply:
Thank you @Omar-Muhamad , we'll do the change.